static/app/components/modals/savedSearchModal/createSavedSearchModal.spec.tsx

```diff
@@ -13,15 +19,21 @@ describe('CreateSavedSearchModal', function () {
   });
 
   const defaultProps = {
-    Header: p => p.children,
-    Body: p => p.children,
-    Footer: p => p.children,
+    Body: ModalBody,
+    Header: makeClosableHeader(jest.fn()),
+    Footer: ModalFooter,
+    CloseButton: makeCloseButton(jest.fn()),
+    closeModal: jest.fn(),
     organization,
     query: 'is:unresolved assigned:lyn@sentry.io',
     sort: IssueSortOptions.DATE,
   };
 
   beforeEach(function () {
+    // XXX: Something here triggers a "Can't perform a React state update on an unmounted component"
+    // This warning will be gone in React 18, remove this when we upgrade.
+    jest.spyOn(console, 'error').mockImplementation(jest.fn);
```

Comment:
does this litter the console output of test runs? I'm trying to avoid us having extranious output in passing tests.

Can you just wrap something in `act`? or add some `await act(tick)`s somewhere?

Reply:
Thanks for calling this out, I was being lazy. Turns out using fake timers was causing the issue.